index.html

```diff
@@ -0,0 +1,243 @@
+            <h3 class="program-heading">Main Program</h3>
+            <span class="underline"></span>
+            <div class="program-outline">
+                <ul class="program">
+                    <li>
+                        <img src="images/icon1.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon3.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon4.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon2.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur</p>
+                    </li>
+                </ul>
+                <ul class="program">
+                    <li>
+                        <img src="images/icon5.png" alt="edu-fair icon">
+                    </li>
+                    <li>
+                        <h3>Lorem ipsum</h3>
+                    </li>
+                    <li>
+                        <p>Lorem ipsum dolor sit amet consectetur.</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="entire-program">
+                <a href="#">SEE THE WHOLE PROGRAM</a>
+            </div>
+            <div class="join-program">
+                <a href="#">CLICK HERE TO KNOW MORE</a>
+            </div>
+        </section>
+        <article class="guest-speakers">
+            <h3>Guest Speakers</h3>
+            <span class="underline"></span>
+            <div id="guest-speakers"></div>
+        </article>
+        <section class="sponsors">
+            <h3>Sponsors</h3>
+            <span class="underline"></span>
+            <div class="sponsors-logo">
+              <img
+                class="sponsor-logo"
+                src="./images/sponsor-one.jpg.jpg"
+                alt="Sponsor logo one"
+              />
```

Comment:
- [ ] [Optional] Please, make sure the alignments and padding as per given design. It's a best practice to follow given design Margin, Padding, Alignments as much as you can.